### DISCUSSION TITLE
Enabling the GitHub Copilot extension in VS Code changes user settings

### DISCUSSION THREAD

**Discussion by mor-arusi-riverside** [community/community#125902] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

After setting the `"github.copilot.editor.enableAutoCompletions"` setting in my user settings to `false`, whenever I turn on the github copilot extension this setting gets reset to `true`, this also happens whenever I reopen vs-code.

An extension should not change user settings when it is enabled.

EDIT:
I later found an issue was opened at around the same time I opened this one for the vscode-copilot-release repo:
https://github.com/microsoft/vscode-copilot-release/issues/1248

**Comment by y-guang** [comment]
Exactly the same. The user `settings.json` line `"github.copilot.editor.enableAutoCompletions" ` is always turned on.

**Comment by tpope** [comment, marked as answer]
Fixed in 1.197.0.

**Comment by aakagi** [comment]
This issue is still happening for me.

I'm on version `v1.207.0`, but it's also forcing me to install `GitHub.copilot-chat` at `v0.16.1`, which might be adding it.

It's especially infuriating how it adds a deprecated setting 🤦

![image](https://github.com/community/community/assets/7625040/87e4c684-7470-432e-8379-697e3d976044)

**Reply by ayim** [reply]
@Akash1134  / @digitarald for routing to the correct repo

**Reply by tpope** [reply]
@aakagi Weird, that definitely shouldn't be happening. Can you tell me if you have any other `github.copilot` options set in your config? And if this is your global settings or workspace settings?
